src/System.Collections/tests/System.Collections.Tests.csproj

```diff
@@ -8,7 +8,7 @@
     <OutputType>Library</OutputType>
     <AssemblyName>System.Collections.Tests</AssemblyName>
     <RootNamespace>System.Collections.Tests</RootNamespace>
-    <NugetTargetMoniker>.NETStandard,Version=v1.3</NugetTargetMoniker>
+    <NugetTargetMoniker>.NETStandard,Version=v1.5</NugetTargetMoniker>
```

Comment:
Some of the System.Reflection members needed for `ObjectCloner` to compile, like `BindingFlags` and `TypeInfo.GetMethod`, were missing in the netstandard1.3 contract assembly.